contrib/conflict_resolution/resolve_interactively.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""Ask user to resolve a vdirsyncer sync conflict interactively.
+
+Needs a way to ask the user.
+The use of https://apps.kde.org/kdialog/ for GNU/Linix is hardcoded.
+
+Depends on python>3.5 and KDialog.
+
+Usage:
+  Ensure the file executable and use it in the vdirsyncer.conf file, e.g.
+
+  conflict_resolution = ["command", "/home/bern/vdirsyncer/resolve_interactively.py"]
+
+This file is Free Software under the following license:
+SPDX-License-Identifier: Apache-2.0
```

Comment:
`vdirsyncer` is BSD-licensed, so including a file under this licence would be problematic since the source thing can't trivially be redistributed together.

Reply:
There are many Free Software sources that are distributed with parts that are under different licenses. Usually that is easily done, and it is under `contrib/`, so I haven't given it much thought. If you fell it is important I can change the license (and first have to check which of the "BSD"-Licenses is actually used and find its SPDX-License-Identifier.. ;) (Note the BSD-Licenses are not ideal, if not for the problem that they mix licensing text and copyright owner notes.)

Reply:
It's the three-clause BSD licence. `SPDX short identifier: BSD-3-Clause`

I'd rather not include things in the repository that are under a different licence, it just make redistribution more complex (redistributors need to keep in that different bits have different licences).

I can include this under an Apache license, but that means that the source can never be reused or merged into `vdirsyncer` in any way, which creates a weird scenario for contributors too.

Reply:
License changed accordingly.